docs/marketplace/dev/front-end/table-design/index.md

```diff
@@ -211,6 +213,8 @@ export interface TableConfig {
     [featureName: string]: TableFeatureConfig | unknown;
 }
 
+export interface TableConfig extends TableCoreConfig {}
```

Comment:
I don't remember why did we create `TableConfig` and `TableCoreConfig` as 2 separate interfaces as `TableConfig` is empty anyway?

Reply:
Discussed/Reverted